dts/arm/atmel/samd21.dtsi

```diff
@@ -42,3 +42,7 @@
 &sercom5 {
 	interrupts = <14 0>;
 };
+
+&adc {
+	interrupts = <23 0>;
+};
```

Comment:
Now that support for samr21 got merged, you can add the same for `samr21.dtsi` too :smiley:

Reply:
ACK, rebased.